### ISSUE TITLE
Hacking attempt

### ISSUE THREAD

**Issue by zxzharmlesszxz** [nilsteampassnet/TeamPass#2465]
### Steps to reproduce
1. install 2.1.27.23
2. up to 2.1.27.27

App not usable - to many errors in your code, got 'you don`t have permissions to see this page', 'hacking attempt' when try to brows folders at start page when i logged in

Fix installer!
`user_ip_lastdate varchar(50) NOT null DEFAULT NULL,` - really?

#### Web server error log
```
This log is empty
```

#### Log from the web-browser developer console (CTRL + SHIFT + i)
```
'you don`t have permissions to see this page'
'hacking attempt'
```

**Comment by CrazyHorse019** [comment]
Hi, we upgraded to version 2.1.27.28 and this issue is also present. Exactly as Matwolf08 describes.

**Comment by nilsteampassnet** [comment]
I don't understand what is happening here.
I've played my test cases as usual and none failed.
I'm also currently using this version without issues.
This occurred just after upgraded to x.28?
No errors during installation?

No structural changes of DB were introduced in x.27 and x.28 so this sound strange to me. I don't have any investigation path.
I would need an access to your server

**Comment by abelmferreira** [comment]
Same with a fresh install, sorry but i open a new issue #2467 without seeing this.

Before a fresh install with 2.1.27.28, i maked a fresh install of 2.1.27.26, because no users table bug of 2.1.27.27, and updated from 2.1.27.26 to 2.1.27.27 without any setting changes and had same issue.

I cant say if this issue appears on 2.1.27.26 too.

**Comment by nilsteampassnet** [comment]
I've answered in #2467

**Comment by CrazyHorse019** [comment]
Hi, I've replaced the upgrade_run_2.1.27.php in the install file with the one you have provided in the development branch. Unfortunately the issue still persists. 

First im presented with a blank information box
![screenshot 2018-11-08 at 09 38 33](https://user-images.githubusercontent.com/4571018/48190415-8db72b00-e33a-11e8-86e4-74426031f07a.png)
Then I try to get one of my passwords and then another information box appears saying an error appears. 
![screenshot 2018-11-08 at 09 38 59](https://user-images.githubusercontent.com/4571018/48190416-8db72b00-e33a-11e8-9fee-b336dcf1f608.png)
as per the above image it states i dont have access to those passwords so then i request it. However it then tells me there that the requested JSON parse failed. 
![screenshot 2018-11-08 at 09 39 22](https://user-images.githubusercontent.com/4571018/48190417-8db72b00-e33a-11e8-8f27-db3fe30262be.png)

I hope this helps.

**Comment by nilsteampassnet** [comment]
Fix provided in 2.1.27.29
